### ISSUE TITLE
Basic Screen Reader Functionality on Fabric

### ISSUE THREAD

**Issue by chiaramooney** [microsoft/react-native-windows#11899]
Add support for key props to unblock basic screen reader navigation and announcements allowing a real AT user to “read” the UI. 

Task  | Description | Status
| -- | -- | --
focusable | Add support for the `focusable` prop on Fabric. Elements should be focusable when prop is set to true and not focusable when prop is set to false. Behavior should change upon fast refresh. | https://github.com/microsoft/react-native-windows/pull/11674
accessible | Add support for the `accessible` prop on Fabric. | https://github.com/microsoft/react-native-windows/pull/11719
role / accessibilityRole | Add support for the `role`/`accessibilityRole` prop on Fabric | https://github.com/microsoft/react-native-windows/pull/11719 https://github.com/microsoft/react-native-windows/pull/11876

IRawElementProviderFragment::SetFocus | Add support for the `accessibilityState` prop on campus | https://github.com/microsoft/react-native-windows/pull/11674
```[tasklist]
### Tasks
- [ ] https://github.com/microsoft/react-native-windows/pull/11674
- [ ] https://github.com/microsoft/react-native-windows/pull/11719
- [ ] https://github.com/microsoft/react-native-windows/pull/11876
```

**Comment by chrisglein** [comment]
Looks like [accessibilityState](https://reactnative.dev/docs/text#accessibilitystate) is the last remaining item on this list. @YajurG @chiaramooney can you split out a tracking issue for this so we can include in planning?
